catalog/imaging/pom.xml

```diff
@@ -28,7 +28,7 @@
     <name>Alliance :: Imaging</name>
 
     <properties>
-        <camel.version>2.16.1</camel.version>
+        <camel.version>2.19.0</camel.version>
```

Comment:
I think that this upgrade proably should have been done as part of the effort [DDF-3006](https://codice.atlassian.net/browse/DDF-3006). @vinamartin and I will make sure that this will  not be an issue with the Karaf version upgrade.